pyProbeParticle/ocl/field.py

```diff
@@ -7,9 +7,9 @@
 import pyopencl as cl
 from pyopencl import array
 
-from .GridUtils import loadCUBE, loadXSF
-from .basUtils import loadAtomsCUBE, loadXSFGeom
-from .fieldFFT import getProbeDensity
+from ..GridUtils import loadCUBE, loadXSF
+from ..basUtils import loadAtomsCUBE, loadXSFGeom
+from ..fieldFFT import getProbeDensity
 
 try:
     from reikna.cluda import ocl_api, dtypes
```

Comment:
does it mean it can run also without reikna? if we run just point-charge electrostatics and don't need FFT ?

Reply:
Correct. I made it this way because I did not want to break some scripts that I had running in environments without reikna. Eventually when we get this to PyPI, we will probably add reikna as a dependency, so could then get rid of this try-except.